requirements.txt

```diff
@@ -1,6 +1,6 @@
 pycodestyle~=2.4
-pytest==3.0.6
+pytest==3.10.1
```

Comment:
This is the most recent in the 3.x gen. I am not willing to introduce breaking changes for now, but latest is already 4.3.0.